src/main/java/com/strandls/mail/consumer/RabbitMQConsumer.java

```diff
@@ -283,6 +287,18 @@ private void processMessage(String message) {
 			case PAGE_DELETED:
 				pageMailService.sendPageDeletedMail(info);
 				break;
+			case DATATABLE_CREATED:
+				dataTableMailService.sendDataTableAddedMail(info);
+				break;
+			case DATATABLE_DELETED:
+				dataTableMailService.sendDataTableDeletedMail(info);
```

Comment:
Pls look into tagged user flow and add support to tagged user for datatable mails as well